server/db.mjs

```diff
@@ -56,20 +56,40 @@ export const deleteTask = async (id) => {
     [id],
   );
 
-  console.log({ deletedTask });
+  window.alert("Deleted Task: ", { deletedTask });
   return deletedTask;
 };
 
 //USER QUERIES
 export const getUser = async (email) => {
-  const user = await db.any("SELECT * FROM users WHERE email = $1", [email]);
-  if (user.length < 1) {
+  const account = await db.any(
+    `SELECT * 
+FROM users
+JOIN settings
+  ON users."id" = settings."userId"
```

Comment:
Tiny thing, it might be better to shift `users."id"` to `users.id`, or consistently stick with one of those, of course unless it's case-sensitive like `"userId"`.